R/addAlpha.R

```diff
@@ -0,0 +1,234 @@
+#' Estimate alpha diversity indices
+#' 
+#' The function estimates alpha diversity indices optionally using rarefaction,
+#' then stores results in \code{\link{colData}}.
+#' 
+#' @param x a \code{\link{SummarizedExperiment}} object.
+#' 
+#' @param assay.type the name of the assay used for calculation of the
+#'   sample-wise estimates (default: \code{assay.type = "counts"}).
+#'   
+#' @param index a \code{character} vector, specifying the alpha diversity 
+#'   indices to be calculated.
+#'   
+#' @param name a name for the column(s) of the colData the results should be
+#'   stored in. By default this will use the original names of the calculated
```

Comment:
We could use @inheritParams for : "assay.type" and "name" arguments from some other functions?